src/index.js

```diff
@@ -10,13 +10,15 @@ class DAPIClient {
    * @param options
    * @param {Array<Object>} [options.seeds] - seeds. If no seeds provided
    * default seed will be used.
-   * @param {number} [options.port] - default port for connection to the DAPI
-   * @param {number} [options.timeout] - timeout for connection to the DAPI
-   * @param {number} [options.retries] - num of retries if there is no response from DAPI node
+   * @param {number} [options.port=3000] - default port for connection to the DAPI
+   * @param {number} [options.nativeGrpcPort=3010] - Native GRPC port for connection to the DAPI
+   * @param {number} [options.timeout=2000] - timeout for connection to the DAPI
+   * @param {number} [options.retries=3] - num of retries if there is no response from DAPI node
    */
   constructor(options = {}) {
     this.MNDiscovery = new MNDiscovery(options.seeds, options.port);
     this.DAPIPort = options.port || config.Api.port;
+    this.nativeGrpcPort = options.nativeGrpcPort || config.grpc.nativePort;
```

Comment:
Why `native` instead of just `port`?

Reply:
Because GRPC has two ports, one for GRPC Native one for GRPC Web.

Reply:
Where is the web port then?

Reply:
For web, we use the same port as JSON RPC.